srcpkgs/clipmenu/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'clipmenu'
 pkgname=clipmenu
-version=5.6.0
-revision=2
+version=6.1.0
+revision=1
 archs=noarch
 depends="bash dmenu"
 short_desc="Clipboard management using dmenu"
-maintainer="Hristos N. Triantafillou <voidpkg@hristos.lol>"
```

Comment:
Is this you? Why is it orphaned?

Reply:
> Is this you? Why is it orphaned?

The last update was in January 2019. I just thought it was appropriate, is there any criteria for that?

Reply:
It would be appropriate to reach out to the maintainer and ask, why the package is not updated.
It's likely the maintainer is not active anymore but there might as well be a valid reason not to update.

Reply:
That makes sense. Thanks for the answer